src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -0,0 +1,128 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.IntervalList;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.GATKTool;
+import org.broadinstitute.hellbender.utils.IntervalUtils;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.apache.commons.math3.util.FastMath;
+import java.io.File;
+import java.util.List;
+
+/**
+ * This tool takes in intervals via the standard arguments of {@link GATKTool} preprocesses them and creates bins that
+ * cover the processed intervals. The intervals are first padded, then they are uniquified and the overlapping intervals
+ * are merged. Finally, the bins are created.
+ *
+ * <p>Standard GATK engine arguments include -L and -XL. For example, for the -L argument, the tool accepts gatk-style
```

Comment:
`GATK-style` (I know you copied this from `SplitIntervals`, so feel free to also make the fix there.  @davidbenjamin would call this an example of the "boy-scout rule", http://pragmaticcraftsman.com/2011/03/the-boy-scout-rule/)

Reply:
👍 👨‍🌾